i18n/ru-RU.js

```diff
@@ -15,11 +14,9 @@ let enUS = {
   "liLaunch"       :"Если всё идёт по плану, пейлоад запустится на приставке!",
   "h1SetupDelivery" :"Настройки отправки пейлоада",
   "h4SelectPayload" :"Выберите пейлоад:",
-  "optionInstantV5" :"Прямая загрузка в CFW (5.0.x)",
-  "optionInstantV4" :"Прямая загрузка в CFW (4.0.x)",
-  "optionHekateV5"  :"Hekate для приставок на 5.0.x",
-  "optionHekateV4"  :"Hekate для приставок на 4.0.x",
+  "optionCTCaerHekate" :"CTCaer's Hekate Mod v3.0 (All Firmwares)",
```

Comment:
Can you change this line to 
`"optionCTCaerHekate" :"CTCaer's Hekate Mod v3.2 (All Firmwares)",`

Reply:
Yep, thanks, I'll do so right now